### DISCUSSION TITLE
Welcome to webgems Discussions!

### DISCUSSION THREAD

**Discussion by lostdesign** [lostdesign/webgems#237] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MWR1** [comment]
Thank you a lot for maintaining and developing this website. It's a genuinely useful resource!

**Comment by topik0** [comment]
This site is great for all webdevs, newbies, experts, or anywhere in between.  Thanks for making this site and maintaining it!
